src/Matcher/Pattern/TypePattern.php

```diff
@@ -69,10 +78,28 @@ public function matchExpanders($value)
     }
 
     /**
-     * @return null|string
+     * {@inheritdoc}
      */
     public function getError()
     {
         return $this->error;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function hasExpander(string $expanderName)
+    {
+        foreach ($this->expanders as $expander) {
+            if (!$this->expanderInitializer->hasExpanderDefinition($expanderName)) {
```

Comment:
wait, why do we need to use expanderInitializer? Can't we check if expander deos not exists in `$this->expanders` field?